velox/connectors/hive/iceberg/PositionalDeleteFileReader.cpp

```diff
@@ -247,18 +247,6 @@ void PositionalDeleteFileReader::updateDeleteBitmap(
         deletePositions[deletePositionsOffset_] - rowNumberLowerBound);
     deletePositionsOffset_++;
   }
-
```

Comment:
For line 152: Remove the comment that "at this point we
  // don't know how many rows the base row reader would scan yet."